### DISCUSSION TITLE
Building and using the new tray module

### DISCUSSION THREAD

**Discussion by rocky0x4e** [polybar/polybar#3015] [Q&A] [answered]
I build from the latest source, and I can see that the tray module was being built.
But when I use it, I always get the error:
```
polybar|error: Disabling module "systray" (reason: Unknown module: internal/tray)
```
Can someone please tell me what did I do wrong?

![image](https://github.com/polybar/polybar/assets/59199575/c0f98a1a-0d02-4419-857f-5a88b228c914)

**Comment by patrick96** [comment, marked as answer]
Most likely you are not actually using the version of polybar you just compiled.

**Reply by rocky0x4e** [reply]
I removed the old version, then built and installed the new one, but still the same error

**Reply by patrick96** [reply]
What is the output of `polybar -vvv`?

**Reply by rocky0x4e** [reply]
```
polybar 3.6.3

Features: +alsa +curl +i3 +mpd +network(wireless-tools) +pulseaudio +xkeyboard

X extensions: +randr (+monitors) +composite +xkb +xrm +xcursor

Build type: Release
Compiler: /usr/bin/g++
Compiler flags:  -O3 -DNDEBUG -Wall -Wextra -Wpedantic -Wsuggest-override
Linker flags:  -Wall -Wextra -Wpedantic -Wsuggest-override  -Wall -Wextra -Wpedantic -Wsuggest-override
```

**Reply by patrick96** [reply]
That's still last release of polybar. If you compiled from master, the version would look something like `polybar 3.6.3-212-g16102c25`

**Reply by rocky0x4e** [reply]
You were right, there was another polybar installed in another location.
I did not remove it completely, my bad.
Thanks a lot for the help
Best wishes to you and the team, awesome work.
